packages/components/layout/src/LayoutHeader.styles.ts

```diff
@@ -8,6 +8,7 @@ export const getLayoutHeaderStyles = (
 ) => ({
   root: css({
     borderBottom: `1px solid ${tokens.gray200}`,
+    padding: `0 ${tokens.spacingL}`,
```

Comment:
As discussed we cannot have padding here, it will need to be handled by the child component to avoid breaking existing headers. Sidebar and body should behave the same to be consistent.

Reply:
removed